ansible/requirements.txt

```diff
@@ -0,0 +1,8 @@
+molecule
+molecule-ec2
+ansible-lint
+boto
+boto3
+ansible==2.9.20
```

Comment:
We use different ansible in `konvoy-image-builder` container https://github.com/mesosphere/konvoy-image-builder/blob/eb4ac11723e7ca19ed7392d923299bd7fa5177d7/Dockerfile#L9

Reply:
thanks a lot yeah makes sense I'll change that and run the tests again